lib/plugins/slack/attachements.js

```diff
@@ -108,9 +108,16 @@ module.exports = function(dataCollection, slackOptions) {
       }
     }
 
+    let message = '';
+    if (resultBaseURL) {
+      const baseDir = storageManager.getRelativeBaseDir();
```

Comment:
I think we should reconsider the approach with relativeBaseDir. It's not a good fit for use with outputFolder. ```--resultBaseURL="http://example.com/bar" --outputFolder="../../foo"``` will result in ```http://example.com/foo/...```when formatted by Slack. ```--resultBaseURL="http://example.com/bar" --outputFolder="/var/www/foo"``` becomes ```http://example.com/bar//var/www/foo``` which is probably not what the user wanted.